src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -290,6 +291,18 @@ private static void addCrossRefRelatedFieldsForAutoComplete(JabRefPreferences pr
         prefs.putStringList(JabRefPreferences.AUTOCOMPLETER_COMPLETE_FIELDS, new ArrayList<>(keys));
     }
 
+    private static void migrateDoNotResolveFields(JabRefPreferences prefs) {
+        Set<String> keys = new LinkedHashSet<>(prefs.getStringList(JabRefPreferences.DO_NOT_RESOLVE_STRINGS_FOR));
```

Comment:
Migrations are run every time, right? So if someone decides to remove say `url` from the exception list, then this migration would readd upon the next start of JabRef, or not?

Reply:
Yes. But ignore this. The whole logic will be inverted.